src/CXXRTL.py

```diff
@@ -48,32 +48,32 @@ def DO_SIM(latency, parser_state):
     f.close()
 
     # Generate main.cpp
-    main_cpp_text = """
+    main_cpp_text = f"""
 // Default main.cpp template
 #include <iostream>
-#include "top.cpp"
+#include "{SYN.TOP_LEVEL_MODULE}.cpp"
 
 // Names generated by PipelineC tool
 #include "pipelinec_cxxrtl.h"
    
 using namespace std;
    
 int main()
-{
-    cxxrtl_design::p_top top;
+{{
```

Comment:
Did an extra close open paren sneak in here?

I dont mind fixing real quick - will do post merge in just a sec

Reply:
Oh is this f string syntax confusing me / maybe my syntax highlighter hah?